tests/BenchmarkDotNet.IntegrationTests/InProcessTest.cs

```diff
@@ -258,6 +270,13 @@ public decimal InvokeOnceValueType()
                 return DecimalResult;
             }
 
+            [Benchmark]
+            public string InvokeOnceReturnParamValue()
```

Comment:
I don't think this extra benchmark method is necessary. Adding the required property should be enough.